source/form.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Form</title>
+  </head>
+  <body>
+    <header class="main-header">
+      <a class="main-header__logo" href="#">
+        <img class="main-header__logo-image" src="#" alt="#" />
+      </a>
+      <a class="main-header__title" href="#">
+        <img class="main-header__title-logo" src="#" alt="#" />
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню.</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav-list">
+            <li class="main-nav-list__item">
+              <a href="index.html">главная</a>
+            </li>
+            <li class="main-nav-list__item">
+              <a href="catalog.html">каталог продукции</a>
+            </li>
+            <li class="main-nav-list__item main-nav-list__item--current">
+              <a href="#">подбор программы</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="form-container">
+      <section class="program-selection">
+        <div class="program-selection__wrapper">
+          <h1 class="program-selection__title">Подбор программы</h1>
+          <p class="program-selection__description">
+            Заполните анкету, и мы подберем программу питания для вашего кота
+          </p>
+          <form
+            class="program-selection-form"
+            action="https://echo.htmlacademy.ru/"
+            method="post"
+          >
+            <label class="program-selection-form__label input-name" for="name-field" id="name">
```

Comment:
```suggestion
            <label class="program-selection-form__label" for="name-field" id="name">
```
input-name это зачем вообще? у тебя все одинаковое 
лейблы отличаются только звездочкой
какой полет фантазии у дизайнера должен быть чтоб сделать все лейблы и инпуты разными?